crates/ln-dlc-node/src/tests/mod.rs

```diff
@@ -143,12 +143,12 @@ impl Node {
     /// the peers involved!! It may not always work.
     async fn open_channel(
         &self,
-        peer: NodeInfo,
+        peer: &Node,
```

Comment:
Was this change really needed given that you make use of `peer.info` anyways?

Reply:
no it wasn't 😅 . I reverted it.